example/spec/parallel/syncGranule/SyncGranuleDuplicateHandlingSpec.js

```diff
@@ -278,18 +277,18 @@ describe('When the Sync Granule workflow is configured', () => {
       it('captures both files', async () => {
```

Comment:
Nit: not your PR but this should probably be 'captures the additional file' - I had to do a quick detour to make sure I understood why it was expecting the files.length to be 3 here.